.vscode/launch.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
The thing specific to your workspace is VS code. Should we check in configurations for other IDEs too?